bin/cli.js

```diff
@@ -32,6 +33,87 @@ program.command("install [firmware]")
     .option("-p, --port <port>", "Serial port board is attached to")
     .option("-f, --firmata [firmata]", "Install firmata version of firmware")
     .option("-i, --address <address>", "Specify I2C address, eg 0x67")
-    .action(interchange.install_firmware.bind(interchange));
+    .option("--interactive", "Interactive mode will prompt for input")
+    .action(function (firmware, opts) {
+
+        if (opts.interactive) {
+
+            interchange.get_ports(function (err, ports) {
+
+                var questions;
+
+                if (err) {
+                    console.error(err);
+                    return;
+                }
+
+                questions = [
+                    {
+                        type: "list",
+                        name: "firmware",
+                        message: "Choose a firmware",
+                        choices: interchange.firmwares.map(function (el) {
+                            return el.name
+                        })
+                    },
+                    {
+                        type: "confirm",
+                        name: "firmata",
+                        message: "Install firmata version?",
+                        default: false
+                    },
```

Comment:
Not sure how to do this with inquirer but in the case of a firmware which ONLY has a firmata version (and we could limit this simply to StandardFirmata for the moment) we probably shouldn't ask if you want to install the firmata firmware because it's implied...

Hopefully that makes sense.